_posts/2014-10-02-sinatra-app-tutorial.markdown

```diff
@@ -203,7 +203,7 @@ Paste the following code into `suffragist.rb`:
 
 {% highlight ruby %}
 get '/results' do
-  @votes = { 'waw' => 7, 'krk' => 5 }
+  @votes = { 'HAM' => 7, 'PIZ' => 5, 'CUR' => 3}
```

Comment:
Can you please add a space before the closing `}` for consistency (and keeping the code in line with RuboCop’s defaults)? :)